docs/operation/solutions.rst

```diff
@@ -0,0 +1,62 @@
+Solutions Guide
+===============
+
+To deploy a Solution in a MetalK8s cluster, a utility script is provided.
+This section describes, step by step, how to deploy a Solution using this
+tool, located at the root of Metalk8s archive:
+
+  .. code::
+
+    /srv/scality/metalk8s-X.X.X/solutions.sh
+
+
+Import a Solution
+*****************
+
+First, the Solution must be imported in the cluster (make the container images
+available through the cluster registry):
+
+  .. code::
+
+    ./solutions.sh import --archive </path/to/solution.iso>
+
+
+Activate a Solution Version
+***************************
+
+Only one version of a Solution can be active at any point in time.
+An active Solution version provides the cluster-wide resources to all other
```

Comment:
`the cluster-wide resources`, for example?

Reply:
CRDs

Reply:
Maybe mention it, i.e: `…provides the cluster-wide resources, such as CRDs, to all other…`

Reply:
done